raysect/core/math/function/function2d/tests/test_base.py

```diff
@@ -137,6 +137,126 @@ def test_pow_function2d_scalar(self):
             r4 = 0 ** self.f1
             r4(-1, 0)
 
+    def test_richcmp_scalar(self):
+        v = [-1e10, -7, -0.001, 0.0, 0.00003, 10, 2.3e49]
+        for x in v:
+            for y in v:
+                ref_value = self.ref1(x, y)
+                greater_value = ref_value + abs(ref_value) + 1
```

Comment:
Being picky here, but could you rename these to higher_value and lower_value as it is easier to interpret.

Reply:
Yep, I can do this